generator/README.md

```diff
@@ -0,0 +1,1186 @@
+# Proxy Library RPC Generator
+
+## Overview
+
+This script provides a possibility to auto-generate Objective-C code (header \*.h and implementation \*.m classes) based on the SDL MOBILE_API XML specification provided as a source of true.
+
+## Requirements
+
+The script requires **Python 3.5** pre-installed on the host system. This is the minimal **Python 3** version that has not yet reached the end-of-life (https://devguide.python.org/devcycle/#end-of-life-branches).
```

Comment:
the code is compatible with 3.5 thus minimum required version is 3.5. This part just notifies that the generator will not work with previous versions. Even after 3.5 will reach EoL, the code will be compatible with that version until somebody puts the incompatible code. Do we still need this fix here?

Reply:
I think the issue that I see here is that if the minimum required version is 3.5, we are _locked_ to that version until a major release of the library. Because 3.5 is close to EOL, bugfixes will not be created for it, and any language features newer than 3.5 cannot be used. In addition, I see no reason to choose 3.5 over a newer, but stable, release like 3.7.6. That gives us additional time to receive bug fixes without making a major version change to the library, and additional language features that can be used in the future if needed.

Reply:
@joeljfischer could the addition "3.5 or higher" fix that `lock` problem? To be honest, I don't think the readme really could lock maintainers to use features of the newest Python version if needed, however, this generator is 3.5 compatible by the fact at the moment.

Reply:
I understand what you're saying. What I'm concerned about is if we say 3.5 or higher, then in a later release we use a version 3.7 part of the language, or rely on a bug fixed in a supported version. In that case, we would have to bump the minimum version, which would break users with Python 3.5 installed, which would be a major version change. We could say something like, "Minimum supported version is 3.7.6. It may work on versions back to 3.5, but this is not supported and may break in the future."

Reply:
@joeljfischer I'm ok with the last suggestion. I will put it in the document. The last stable release of Python is 3.8.2, are you ok with this version in the document?

Reply:
Bilal and I were talking about that, I think we're looking for something a little more stable than 3.8.2, since there have been a few quick patches since the 3.8.0 release. Bilal and I agreed on 3.7.6.

Reply:
@joeljfischer done in https://github.com/smartdevicelink/sdl_ios/pull/1556/commits/50a2d3ea9e7bd7f95cc8f693856ef7451085f736